activity.js

```diff
@@ -29,9 +29,15 @@ class Activity {
       timer.innerText = `${this.minutes}:${this.seconds}`;
     }
   }
+
   markComplete() {
+    this.completed = true;
   }
 
   saveToStorage() {
+    activityCards.push(this)
+    localStorage.setItem('activityCards', JSON.stringify(activityCards));
+    pastActivityCards.innerHTML = '';
+    showMyLog();
   }
```

Comment:
This function has been refactored as a class method, invoked at main.js line 169. However, there's still some redundancy in these functions that could possibly be re-factored.